### DISCUSSION TITLE
Component "PhotoUploader": Allow tag pictures

### DISCUSSION THREAD

**Discussion by jcatdom** [SUI-Components/sui-components#2049] [Ideas and new features]
**Description**
There's a need to create a new variation of the component PhotoUpload, allowing to tag pictures.

**Discussion and actions**

- Should it be a different component or a new variation of the existing one?  Confirmed, a new variation.
- For tagging, should it be a Select or a Dropdown List?
- For tagging, should it be single o multiple choice?
- Max number of items to show in the Select/Dropdown

**Specs**
![PhotoUploader-Tag-Pictures](https://user-images.githubusercontent.com/94112525/157255451-52a130e6-27d9-45b1-88ff-a358c6ee4166.png)

**Comment by andresin87** [comment]
https://github.com/SUI-Components/sui-components/pull/2105
